common/views/components/OpeningHoursTable/OpeningHoursTable.js

```diff
@@ -0,0 +1,29 @@
+import {spacing, font} from '../../../utils/classnames';
+
+type Props = {|
+  id?: string,
+  place: any, // TODO
```

Comment:
Given that a lot of this is quite complicated, we should probably try to type these before merging.